### ISSUE TITLE
Freetype error while compiling

### ISSUE THREAD

**Issue by bladeoner** [dborth/snes9xgx#731]
@dborth this is the error which I'm getting while compiling with the current Freetype:

linking ... snes9xgx-wii.elf
/opt/devkitpro/portlibs/ppc/lib/libfreetype.a(ftbzip2.o): In function `ft_bzip2_stream_close':
ftbzip2.c:(.text+0x3c): undefined reference to `BZ2_bzDecompressEnd'
/opt/devkitpro/portlibs/ppc/lib/libfreetype.a(ftbzip2.o): In function `ft_bzip2_file_fill_output':
ftbzip2.c:(.text+0x110): undefined reference to `BZ2_bzDecompress'
/opt/devkitpro/portlibs/ppc/lib/libfreetype.a(ftbzip2.o): In function `ft_bzip2_stream_io':
ftbzip2.c:(.text+0x3bc): undefined reference to `BZ2_bzDecompressEnd'
ftbzip2.c:(.text+0x3f4): undefined reference to `BZ2_bzDecompressInit'
/opt/devkitpro/portlibs/ppc/lib/libfreetype.a(ftbzip2.o): In function `FT_Stream_OpenBzip2':
ftbzip2.c:(.text+0x668): undefined reference to `BZ2_bzDecompressInit'
collect2: error: ld returned 1 exit status
/opt/devkitpro/devkitPPC/wii_rules:24: recipe for target '/opt/devkitpro/examples/snes9xgx-master/executables/snes9xgx-wii.elf' failed
make[2]: *** [/opt/devkitpro/examples/snes9xgx-master/executables/snes9xgx-wii.elf] Error 1
Makefile.wii:110: recipe for target 'build_wii' failed
make[1]: *** [build_wii] Error 2
make[1]: Leaving directory '/opt/devkitpro/examples/snes9xgx-master'
Makefile:10: recipe for target 'wii' failed
make: *** [wii] Error 2

It seems it uses a specific function which is not in the latest versions of Freetype.

**Comment by dborth** [comment]
If you add  -lbz2 it will compile
...but when I compile with the latest freetype posted in the devkit portlibs, the emus crash as soon as a wiimote connects. So I left the checked in version without  -lbz2

**Comment by revvv** [comment]
My private build with latest devkitpro runs successfully on my Wii. (I want to add USB controllers.)

**Comment by arthurfreitas96** [comment]
@dborth This is the first time I'm compiling snes9xgx and I'm facing some issues. I was experiencing extreme laggy output from my 3rd party controller and after testing some previous releases of snes9xgx, I found this patch made by matguitarist using a modded libogc: https://www.mediafire.com/FIX94. It worked for me and the laggy behavior was gone.
However, this patch was made using snes9xgx's version 4.3.2 and I'm trying to compile the current release of the emulator using the modded libogc and portlibs folders provided by matguitarist in his post. I installed devkitPro in my PC (currently using Windows 10) and ran the command 'make -f Makefile' in the snes9xgx-4.4.8 folder. After getting some errors, I followed Wiipathy advice (here: https://github.com/Fledge68/WiiFlow_Lite/issues/52) and changed my DevkitPPC to r29.1. Now I'm stuck with the error '/bin/sh: freetype-config: command not found'. What am I missing?
I found a youtuber reviewing an off-brand controller and he experienced the same laggy output, here: https://www.youtube.com/watch?v=71o1M7GLdaQ&list=PLqdSLlkg_8NsTqbqcGGxqP-SiExB7HGwX.
Thank you in advance.
